client/app/dns-zone/dns-zone.controller.js

```diff
@@ -71,21 +68,18 @@ angular.module('App').controller(
 
     loadDomain() {
       this.loading.domainsInfos = true;
-
-      this.$q
-        .all({
-          product: this.WucProducts.getSelectedProduct(true),
-          tabZone: this.Domain.getTabZoneDns(this.$stateParams.productId, 1),
-        })
-        .then(({ product, tabZone }) => {
+      this.Domain.getTabZoneDns(this.$stateParams.productId, 1)
```

Comment:
Could `1` param be stored in a const please ? In order to see directly what is the purpose of this param